pkg/fleet-manager/pipeline/render/rbac.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright Kurator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package render
+
+import (
+	"fmt"
+	"io/fs"
+)
+
+const (
+	// RBACTemplateFileName is the name of the RBAC template file.
+	RBACTemplateFileName = "rbac.tpl"
+	RBACTemplateName     = "pipeline rbac template"
+)
+
+// RBACConfig contains the configuration data required for the RBAC template.
+// Both PipelineName and PipelineNamespace are required.
+type RBACConfig struct {
+	PipelineName      string // Name of the pipeline.
+	PipelineNamespace string // Kubernetes namespace where the pipeline is deployed.
+}
+
+// ServiceAccountName generates the service account name using the pipeline name and namespace.
+func (rbac RBACConfig) ServiceAccountName() string {
+	return rbac.PipelineName
+}
+
+// RoleBindingName generates the role binding name using the service account name.
+func (rbac RBACConfig) RoleBindingName() string {
+	return rbac.ServiceAccountName()
+}
+
+// ClusterRoleBindingName generates the cluster role binding name using the service account name.
+func (rbac RBACConfig) ClusterRoleBindingName() string {
+	return rbac.ServiceAccountName()
```

Comment:
Sorry just note this is cluster scope resource, if we create multiple clusterrolebinding, they conflict with same name.

Option one is to make it with different name, alternatively add different sa within the same clusterrolebinding.